elements/lisk-transactions/src/utils/format.ts

```diff
@@ -35,7 +35,7 @@ export const convertBeddowsToLSK = (beddowsAmount?: string): string => {
 	const floating =
 		Number(beddowsAmountBigInt % BigInt(FIXED_POINT)) / FIXED_POINT;
 	const floatingPointsSplit = floating
-		.toLocaleString(undefined, {
+		.toLocaleString('en-US', {
```

Comment:
@shuse2 could you give a scenario where this could make a difference, i tried this and i dint make any difference with undefined
```
(1.2e-7).toLocaleString('en-US', {maximumFractionDigits: 8}) === (1.2e-7).toLocaleString(undefined, {maximumFractionDigits: 8})
```
I was trying to think a test scenario for this fix.

Reply:
I think it's pretty hard to change that behavior. It is looking at local locale, and it is the node.js internal behavior. 
As long as `en-US` works, it should be ok IMO

Reply:
ok, I was trying to understand the problem through a test, ok no problem 👍